common/lib/xmodule/xmodule/course_module.py

```diff
@@ -179,6 +179,12 @@ class CourseFields(object):
                  default=DEFAULT_START_DATE,
                  scope=Scope.settings)
     end = Date(help="Date that this class ends", scope=Scope.settings)
+    cosmetic_display_price = Integer(
+        display_name=_("Cosmetic Course Display Price"),
+        help=_("The cost displayed to students for enrolling in the course. If a paid course registration price is set by platform staff in the database, that price will be displayed instead of this one."),
+        default=0,
+        scope=Scope.settings,
+    )
```

Comment:
what if people want to display a fractional dollar amount - say - $99.99. Unfortunately, our course_modes 'min_price' data model also is an Integer, but it might be good to break that habit if we're just doing a display price